dashboard/views.py

```diff
@@ -61,6 +61,8 @@ def get(self, request, username, *args, **kargs):  # pylint: disable=unused-argu
                     status=exc.http_status_code,
                     data={'error': str(exc)}
                 )
+            except Exception:  # pylint: disable=broad-except
```

Comment:
In case edX is truly down, the python social auth is rising a custom exception (something related to not being able to establish a connection).
In any case if the refresh of the token fails for any reason that is not a 400/401 we need to not raise if we want to keep serving the dashboard.